source/routes.ts

```diff
@@ -130,4 +130,11 @@ export class RouteManager {
       ],
     });
   }
+
+  /**
+   * Find a route by path.
+   */
+  findRouteByPath(path: string) {
+    return this.routes.find((route) => route.path === path);
```

Comment:
We have `ramda` as a dependency, what do you think about using `this.routes.find(propEq('path', path));` instead?

Reply:
Not sure we should keep `ramda` as a dependency.

Reply:
Nice idea.